src/components/Toast/Toast.st.css

```diff
@@ -0,0 +1,60 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color;
+}
+
+:vars {
+    successColor: #57C487;
+    errorColor: #F64D43;
+    statusColor: #212121;
+}
+
+.root {
+    -st-states: rtl, mobile, skin(enum(success, error, status));
+    display: inline-flex;
+    color: #FFFFFF;
+    border-radius: 2px;
+    font-size: 14px;
+    font-family: "HelveticaNeueW01-55Roma";
+    box-shadow: 0 0 4px 0 rgba(0,0,0,0.10), 0 4px 8px 0 rgba(0,0,0,0.10);
+}
+
+.root:skin(success) {
+    background-color: value(successColor);
+}
+
+.root:skin(error) {
+    background-color: value(errorColor);
+}
+
+.root:skin(status) {
+    background-color: value(statusColor);
+}
+
+.gapBeforeMessage,
+.gapAfterMessage {
+    display: inline-block;
+    min-width: 12px;
+}
+
+.message {
+    margin: 16px 20px;
+    display: inline-block;
+    max-height: 40px;
+    line-height: 20px;
```

Comment:
Should we add multiline ellipsis here if the text is longer then 2/3 lines (desktop / mobile)?
It isn't very easy but possible - what do you think?

Reply:
only if there's a css solution for it.
The editor doesn't allow us to us javascript for things like this.

Reply:
You can review an example [here](http://hackingui.com/front-end/a-pure-css-solution-for-multiline-text-truncation/) or [here](https://css-tricks.com/multi-line-truncation-with-pure-css/)
but it works for ltr, we need to test if it will work for rtl

Reply:
Yes, it might not be the most elegant solution, but possible. I can do this. But when I talk with @jonathanadler, he said just cut.

Reply:
ok, i'll do this only with css

Reply:
Sorry I didn't reply, I'm not sure we can use these solutions because they change the styling of the text (justify).
I'll need to check with the designers about this.

Reply:
So yeah, I checked with the designers, and we shouldn't use this solution.
Sorry for not getting back to you earlier.
TPA's using this component should know not to add more than 2 rows of text in the toast.

Reply:
reverted